src/Controller/MemberController.php

```diff
@@ -39,6 +41,21 @@ public function showOneEventById(Event $event): Response
         ]);
     }
 
+    /**
+     * @param UserRepository $userRepository
+     * @Route("/page/{id}", name="member_page")
+     * @return Response
+     */
+    public function index(UserRepository $userRepository, User $user): Response
```

Comment:
ici enleve {id} des param de route
enlève les param de la méthode (tu dois juste avoir index()
et pour récup le user, tu fais un $user=$this->getUser() qui va recup la personne actuellement connectée

Reply:
ok c'est fait :)